schemas/health/cy/phm_0001.json

```diff
@@ -838,7 +838,7 @@
                                 "id": "question23530584-48bc-45c5-bb02-e662a5516b50",
```

Comment:
Would this have errored if you didn't regenerate the CY schema?

Reply:
Hi Mebin can you expand on this comment some more? 

I generated the `cris_0001.json` schema by running the `make translate-schemas` command in the README file after adding the `.po` file into the repo.